core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/queue/MemoryQueue.scala

```diff
@@ -456,6 +456,9 @@ class MemoryQueue(private val etcdClient: EtcdClient,
 
     // This is not supposed to happen. This will ensure the queue does not run forever.
```

Comment:
@style95 I don't think this is actually true since there is no `context.parent ! queueRemovedMsg` when the `Idle` state times out which is fairly frequent behavior and thus `QueueRemoved` is never sent to the manager until `Removed` state times out which wasn't guaranteed.